error[E0603]: module `to_kana` is private

If I try to use this crate, the following error is shown:


```
error[E0425]: cannot find function `to_romaji` in this scope
  --> src/handlebars_util.rs:74:18
   |
74 |     let romaji = to_romaji(&v);
   |                  ^^^^^^^^^ not found in this scope
   |
help: use the `.` operator to call the method `ConvertJapanese::to_romaji` on `&str`
   |
74 |     let romaji = (&v).to_romaji();
   |                  ~  ~~~~~~~~~~~~~
```

Sorry, this is more like Rust newbie question, but I report this just in case. 



- wana_kana = "4.0.0"
- edition = "2024"
- rust-version = "1.85.0"

Yes, but the below attempts did not help:

```rust
handlebars_helper!(katakana_2_romaji: |v: String| {
    let romaji = to_romaji(v.as_str());
..
    let romaji = &v.to_romaji();
..
    let romaji = v.as_str().to_romaji();
```

Though error message is different:
```
error[E0425]: cannot find function `to_romaji` in this scope
```

--- a/README.md
+++ b/README.md
@@ -18,7 +18,6 @@ wana_kana = "4.0"
  use wana_kana::to_kana::*;
  use wana_kana::to_hiragana::*;
  use wana_kana::Options;
- use wana_kana::ConvertJapanese;
  assert_eq!(to_romaji("ワナカナ"), "wanakana");
  assert_eq!(to_hiragana("WanaKana"), "わなかな");
  assert_eq!(to_kana("WANAKANA"), "ワナカナ");
@@ -28,7 +27,7 @@ wana_kana = "4.0"
  
  ```rust
  use wana_kana::ConvertJapanese;
- let foo = "ワナカナ" ;
+ let foo = "ワナカナ";
  assert_eq!(foo.to_romaji(), "wanakana");
  ```
 

--- a/README.md
+++ b/README.md
@@ -13,14 +13,23 @@ wana_kana = "4.0"
 
  Utility library for checking and converting between Japanese characters - Hiragana, Katakana - and Romaji (Ported from https://github.com/WaniKani/WanaKana V4.0.2)
  ## Examples
- ```
+ ```rust
  use wana_kana::to_romaji::*;
  use wana_kana::to_kana::*;
  use wana_kana::to_hiragana::*;
  use wana_kana::Options;
+ use wana_kana::ConvertJapanese;
  assert_eq!(to_romaji("ワナカナ"), "wanakana");
  assert_eq!(to_hiragana("WanaKana"), "わなかな");
  assert_eq!(to_kana("WANAKANA"), "ワナカナ");
+ ```
+
+ or 
+ 
+ ```rust
+ use wana_kana::ConvertJapanese;
+ let foo = "ワナカナ" ;
+ assert_eq!(foo.to_romaji(), "wanakana");
  ```
 
 ## Tests

--- a/README.md
+++ b/README.md
@@ -13,23 +13,14 @@ wana_kana = "4.0"
 
  Utility library for checking and converting between Japanese characters - Hiragana, Katakana - and Romaji (Ported from https://github.com/WaniKani/WanaKana V4.0.2)
  ## Examples
- ```rust
- use wana_kana::to_romaji::*;
- use wana_kana::to_kana::*;
- use wana_kana::to_hiragana::*;
- use wana_kana::Options;
- assert_eq!(to_romaji("ワナカナ"), "wanakana");
- assert_eq!(to_hiragana("WanaKana"), "わなかな");
- assert_eq!(to_kana("WANAKANA"), "ワナカナ");
- ```
-
- or 
  
- ```rust
- use wana_kana::ConvertJapanese;
- let foo = "ワナカナ";
- assert_eq!(foo.to_romaji(), "wanakana");
- ```
+```rust
+use wana_kana::ConvertJapanese;
+
+assert_eq!("ワナカナ".to_romaji(), "wanakana");
+assert_eq!("WanaKana".to_hiragana(), "わなかな");
+assert_eq!("WANAKANA".to_kana(), "ワナカナ");
+```
 
 ## Tests
 

